frontend/src/Dashboard/CrossingListPage/CrossingListItem/CrossingListItem.js

```diff
@@ -124,7 +124,7 @@ class CrossingListItem extends React.Component {
         <div className="CrossingListItemContainer">
           <div className={this.isDirty() ? "DirtyBorder" : ""}>
           <div className="CrossingListItemFlexContainer">
-            <div className="CrossingName">{crossing.name}</div>
+            <a href={'/dashboard/crossing/'+crossing.id} className="CrossingName">{crossing.name}</a>
```

Comment:
es6 string concatenation would be nice here: 
```jsx
<a href={`/dashboard/crossing/${crossing.id}`}...
```

Reply:
good call